docs/Style-Guide.md

```diff
@@ -80,13 +80,18 @@ One of the main rules of formal writing such as reference documentation, or API
 
 **Example**
 
-Less Like this: You can use the following recommendation as an example.
+Less like this: You can use the following recommendation as an example.
 
 More like this: As an example, the following recommendations should be referenced.
 
 To view a live example, refer to the [Decorators.md](/docs/Decorators.md) reference document. 
 
 
+### Avoid using contractions
+
+Contraction are the shortened version of written and spoken forms of a word, i.e. using "don't" instead of "do not".
+Avoid contractions to provide a more formal tone.
```

Comment:
I don't completely agree with that. Before adding this recommendation we should do it in the entire docs. But, regarding _"Avoid using contractions"_, it would be hard to keep consistent the documentation to follow this _rule_, some PR just will pass without being noticed.

Reply:
@RafaelGSS this style guide targets the reference documentation, e.g. the decorators and encapsulation docs. A lot of our current documentation is written in an informal guide style. We need to solve https://github.com/fastify/website/pull/215 so that we can move to splitting up these two styles.

Reply:
Got it!